s-express/src/test/scala/org/ensime/sexp/SexpSpec.scala

```diff
@@ -40,6 +42,12 @@ class SexpSpec extends EnsimeSpec {
     }
   }
 
+  it should "support a list with 1000000 elements" in {
+    val data = List.fill(1000000)(SexpChar('a'))
+    val res = Try(SexpList(data))
+    res shouldBe a[Success[Sexp]]
```

Comment:
the CI doesn't like this line due to erasure. maybe do a pattern match on the Try?